src/views/server.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import ReactDOMServer from 'react-dom/server'
+import App from './containers/App'
+import { createStore } from 'redux'
+import expressoApp from './reducers'
+import { Provider } from 'react-redux'
+import {
+    ExpressoData,
+    ExpressoCall,
+    ExpressoUnderlyingCall,
+    ExpressoResponse,
+    ExpressoRequest
+} from './components/definitions'
+import {
+    ExpressoHttpInterceptorData,
+    ExpressoHttpInterceptorResponse,
+    ExpressoHttpInterceptorRequest
+} from '../interceptors/definitions'
+
+export const handleRender = (
+    originalCall: ExpressoCall,
+    underlyingCalls: ExpressoHttpInterceptorData[]
+): any => {
+    // Grab the initial state from our Redux store
+    const preloadedState = getInitialState(underlyingCalls, originalCall)
+
+    // Create a new Redux store instance
+    const store = createStore(expressoApp, preloadedState)
+
+    // Render the component to a string
+    const html = ReactDOMServer.renderToString(
+        <Provider store={store}>
+            <App data={preloadedState} />
+        </Provider>
+    )
+
+    // Grab the initial state from our Redux store
+    const finalState = store.getState()
+
+    // Send the rendered page back to the client
+    return renderFullPage(html, finalState)
+}
+
+export const renderFullPage = (html: any, preloadedState: any): any => {
+    return `
+    <!doctype html>
+    <html>
+      <head>
+        <title>Redux Universal Example</title>
```

Comment:
Maybe we could change that title (for eg. with stg like "Welcome to expresso debugger")

Reply:
Done

Reply:
Good